ansible/group_vars/nimbus.prater.yml

```diff
@@ -74,31 +74,31 @@ nodes_layout:
     - { branch: 'libp2p',                             build_freq: '*-*-* 19:00:00' }
 
   'metal-02.he-eu-hel1.nimbus.prater': # 1 each
-    - { branch: 'stable',   start: 20000, end: 20001, build_freq: '*-*-* 13:00:00' }
-    - { branch: 'testing',  start: 20001, end: 20002, build_freq: '*-*-* 15:00:00', public_api: true }
-    - { branch: 'unstable', start: 20002, end: 20003, build_freq: '*-*-* 17:00:00' }
-    - { branch: 'libp2p',   start: 20003, end: 20004, build_freq: '*-*-* 19:00:00' }
+    - { branch: 'stable',   start: 18000, end: 18001, build_freq: '*-*-* 13:00:00' }
+    - { branch: 'testing',  start: 18001, end: 18002, build_freq: '*-*-* 15:00:00', public_api: true }
+    - { branch: 'unstable', start: 18002, end: 18003, build_freq: '*-*-* 17:00:00' }
+    - { branch: 'libp2p',   start: 18003, end: 18004, build_freq: '*-*-* 19:00:00' }
 
   'metal-03.he-eu-hel1.nimbus.prater': # 10 each
-    - { branch: 'stable',   start: 20004, end: 20014, build_freq: '*-*-* 11:00:00' }
-    - { branch: 'testing',  start: 20024, end: 20034, build_freq: '*-*-* 15:00:00' }
-    - { branch: 'unstable', start: 20014, end: 20024, build_freq: '*-*-* 13:00:00', open_libp2p_ports: false }
-    - { branch: 'libp2p',   start: 20034, end: 20044, build_freq: '*-*-* 17:00:00' }
+    - { branch: 'stable',   start: 18004, end: 18014, build_freq: '*-*-* 11:00:00' }
+    - { branch: 'testing',  start: 18024, end: 18034, build_freq: '*-*-* 15:00:00' }
+    - { branch: 'unstable', start: 18014, end: 18024, build_freq: '*-*-* 13:00:00', open_libp2p_ports: false }
+    - { branch: 'libp2p',   start: 18034, end: 18044, build_freq: '*-*-* 17:00:00' }
```

Comment:
I now have PTSD, could we keep the range sorted? :sweat_smile: 

ditto for metal-04 & metal-05

Reply:
Sure.